libxbrzscale.cpp

```diff
@@ -18,22 +18,35 @@
 
 #include "libxbrzscale.h"
 
+#include <cstdio>
+#include <iostream>
+
+#ifndef XBRZLIB_RELATIVEPATHSDL
 #include <SDL2/SDL_endian.h>
 #include <SDL2/SDL_error.h>
 #include <SDL2/SDL_pixels.h>
 #include <SDL2/SDL_surface.h>
-#include <cstdio>
+#else
+#include "SDL_endian.h"
+#include "SDL_error.h"
+#include "SDL_pixels.h"
+#include "SDL_surface.h"
+#endif
 
 #include "xbrz/xbrz.h"
 
-//#include <cstdio>
-//#include <cstdint>
-//#include "SDL.h"
-//#include "SDL_image.h"
-//#include "xbrz/xbrz.h"
-
 bool libxbrzscale::bEnableOutput=false;
 
+bool libxbrzscale::bDbgMsg=false;
+
+bool libxbrzscale::bUseCache=false;
+
+bool libxbrzscale::bFreeInputSurfaceAfterScale=true;
+bool libxbrzscale::bFreeOutputSurfaceAfterScale=true;
+
+#ifndef XBRZLIB_NOINLINEGETSETPIX //this may be required to let some compillers linking actually work w/o error: "undefined reference to" these
+inline
+#endif
 Uint32 libxbrzscale::SDL_GetPixel(SDL_Surface *surface, int x, int y)
```

Comment:
Since those functions don't touch any instance properties or methods, they don't need to be inside the class.

I think inline class methods should have implementation in the header if the class declaration is there. It is possible this caused IVAN build issue.

Reply:
I can try it. I upload a commit and travisci compile ivanWin (that was failing cuz of inline on cpp file).